src/lib/tray.py

```diff
@@ -293,7 +293,7 @@ def __init__(self, iconfile):
         # Definition of menus titles
         self.menu = QMenu()
         self.menu_count = QAction(_("Arch-Update"))
```

Comment:
here too

Reply:
This is simply the name of the function that triggers an Arch-Update run, this is not shown to users.  

As said in [this comment](https://github.com/CachyOS/cachy-update/pull/1#discussion_r2226172978), I've been told that the aim was to keep changes as slight as possible by just changing the icon and the "user facing" elements to match "Cachy-Update" (rather than a full rebrand). Is it not the case finally?